### PR TITLE
[TwigBundle] Use vector images on the exception page

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Branch? | master |
| New feature? | yes |
| Tests pass? | yes |
| Fixed tickets | #19756 |
| License | MIT |

Before:
capture du 2016-08-29 15-40-37

After:
capture du 2016-08-29 21-00-36

Ghost SVG provided by  @patrick-mcdougle, thanks to him.

### CODE REVIEW

**Comment by stof** [comment]
:+1:

**Comment by pborreli** [comment]
👍 for the svg ghost
👍 for the removal of svg for quotes
👎 for the style change of quotes (maybe try Arial safe font ?)

File: src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig [hunk 1]
```diff
@@ -4,12 +4,10 @@
...
+            {{ include('@Twig/Exception/exception.svg') }}
         </div>
         <div class="text-exception">
-            <div class="open-quote">
-                <img alt="" src="data:image/gif;base64,R0lGODlhHAAWAMQQANra2+bl5s3Mzevr6/Pz8+jo6O3t7fHx8c/Oz+Pj49PS093d3djX2NXV1eDf4MrJyvb29gAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAABAALAAAAAAcABYAQAWWICSOEDE4AamqRuAsT5yu6hA/wNrcfNysjl5PBOAJAAUDDRLoNRKDndAHnN6k058qaH2QuNelqCAYIm45MfGmIJCkAvUIPNB1td/uAyvEz/UqB0VUagQOZTEjgzx+Kk1CEAU8DAdqB4gPCHVjNwhucphKbzefamAFdlaNEGBZd1V3r1t6fE6wqrJ5XS4Ovb69MyQnv8QhADs=" />
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception.html.twig]: @pborreli this is not an svg quote but a gif, beurk effect :)

File: src/Symfony/Bundle/FrameworkBundle/Resources/public/css/exception.css [hunk 2]
```diff
@@ -56,14 +56,17 @@
 }
 .sf-reset .text-exception .open-quote,
 .sf-reset .text-exception .close-quote {
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Resources/public/css/exception.css]: @pborreli arial is nice :)
capture du 2016-08-29 15-59-54

**Comment by dunglas** [comment]
👍

**Comment by fabpot** [comment]
On small detail that needs to be fixed before merging: changing the Symfony logo size back to the original one.

**Comment by nicolas-grekas** [comment]
Which one is that, in svg?

**Comment by fabpot** [comment]
The only one you changed, the one in the top left corner (as you can see in your before/after screenshots, the new one is smaller and not at the same place).

**Comment by nicolas-grekas** [comment]
I took the preferred logo from here:
http://symfony.com/logo
I'll change for the alt. version then no pb.

**Comment by nicolas-grekas** [comment]
logo updated

**Comment by robfrawley** [comment]
Looks good!

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**